icons/scan-qr-code.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "jguddas",
+    "vexkiddy"
+  ],
+  "tags": [
+    "barcode",
+    "scan",
+    "qrcode",
+    "url",
+    "information",
+    "digital",
+    "scanner"
+  ],
+  "categories": [
+    "development"
```

Not sure if this is the correct category. This not really associated with development in general
I think these categories fit better:
```suggestion
    "account",
    "shopping",
    "devices",
    "security"
```